csrankings-f.csv

```diff
@@ -365,7 +366,6 @@ Flávio de Oliveira Silva,UFU,http://www.facom.ufu.br/~flavio,SW_VMq4AAAAJ
 Flavius Gruian,Lund University,http://www.lunduniversity.lu.se/lucat/user/2b6e0dd31045e02d96f26f40904362d9,SafG-QUAAAAJ
 Flemming Nielson,DTU,https://www.compute.dtu.dk/english,NOSCHOLARPAGE
 Flemming Stassen,DTU,https://www.compute.dtu.dk/english,NOSCHOLARPAGE
-Flora D. Salim,RMIT University,http://florasalim.com,Yz35RSYAAAAJ

```

Comment:
All aliases to this person must be updated (see Flora Salim, Flora Dilys Salim).